pastehunter.py

```diff
@@ -24,13 +24,15 @@
 VERSION = 1.0
 
 # Setup Default logging
-logger = logging.getLogger('pastehunter')
-logger.setLevel(logging.INFO)
+root = logging.getLogger()
 ch = logging.StreamHandler()
 ch.setLevel(logging.DEBUG)
 formatter = logging.Formatter('%(levelname)s:%(filename)s:%(message)s')
 ch.setFormatter(formatter)
-logger.addHandler(ch)
+root.addHandler(ch)
+
+logger = logging.getLogger('pastehunter')
+logger.setLevel(logging.INFO)
```

Comment:
I'm a little confused about this change, honestly. What was the intent here?

Reply:
I tried to clarify in the commit comment.

```
Some frameworks, like Splunk, check to see if a root logger is set and if not then they proceed to modify the logging. Without this patch including the splunk sdk would cause double logging to happen. This change should simply be setting up a logging hierarchy with the root logger actually configured. There should be no difference to the output or configuration of the pastehunter logging
```

Does this help clarify?

Reply:
Huh, interesting. Well, I verified it doesn't seem to impact existing log output.